app/controllers/miq_ae_class_controller.rb

```diff
@@ -2014,8 +2013,8 @@ def fields_get_form_vars
         return
       end
       new_fields = {}
-      field_attributes.each_with_object({}) { |field| field = field.to_sym
-      new_fields[field] = @edit[:new_field][field]}
+      field_attributes.each_with_object({}) { |field| field
```

Comment:
@h-kataria can you clarify this line, where does the code block end. Rubocop is also complaining about it.